MaxText/configs/base.yml

```diff
@@ -65,11 +65,11 @@ decoder_block: "default" # which style of DecoderBlock to use.
 # base_mlp_dim, base_num_decoder_layers and/or head_dim.
```

Comment:
Overall, looks very strong, thank you!.

There are nits below. But more substantively, to merge this, we need a numerical test showing GPU Flash Attention works within numerical tolerence, similar (and possibly exactly the same) as one of the Attention_test.py tests but marked to run on GPU. Let's discuss off-thread if you want to do this or you'd ask a Googler to do it for you.